### ISSUE TITLE
Spyder error on start-up

### ISSUE THREAD

**Issue by MNRK01** [spyder-ide/spyder#8189]
### What steps will reproduce the problem?

I recently installed Anaconda 5.3.0, 64-bit. Upon installation, I tried to run Spyder but this box pops up with the error:

```
TypeError: _start_server() missing 1 required positional argument: 'port'.
```

The console opens up and ipython runs fine. I'm on Win 10.

### Traceback
```python-traceback
TypeError: _start_server() missing 1 required positional argument: 'port'
```

## Versions

* Spyder version: 3.3.1 
* Python version: 3.7.1
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
pyflakes >=0.6.0  :  2.0.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.2.0 (OK)
sphinx >=0.6.6    :  1.8.1 (OK)
rope >=0.9.4      :  0.11.0 (OK)
jedi >=0.9.0      :  0.13.1 (OK)
nbconvert >=4.0   :  5.3.1 (OK)
pandas >=0.13.1   :  0.23.4 (OK)
numpy >=1.7       :  1.15.3 (OK)
sympy >=0.7.3     :  1.3 (OK)
cython >=0.21     :  0.29 (OK)
qtconsole >=4.2.0 :  4.4.2 (OK)
IPython >=4.0     :  7.1.1 (OK)
matplotlib >=2.0.0:  3.0.1 (OK)
pylint >=0.25     :  2.1.1 (OK)
```

**Comment by ccordoba12** [comment]
This an issue with Python 3.7 and it'll be fixed in our next version (3.3.2). In the meantime, please use Python 3.6

**Comment by MNRK01** [comment]
Looking forward to Spyder version 3.3.2. Thanks!
